.vscode/settings.json

```diff
@@ -0,0 +1,36 @@
+{
+    "java.configuration.updateBuildConfiguration": "automatic",
+    "java.server.launchMode": "Standard",
+    "eslint.workingDirectories": [ "./client" ],
```

Comment:
I think this line is what we needed to make that TS file problem go away. This is what's telling ESLint to treat `./client` as the top level of the JavaScript/TS project.